tests/validation/tests/v3_api/test_kdm_changes.py

```diff
@@ -0,0 +1,134 @@
+from copy import deepcopy
+import os
+import pytest
+import requests
+from .common import CATTLE_API_URL
+from .common import create_config_file
+from .common import CATTLE_TEST_URL
+from .common import ADMIN_TOKEN
+from .common import get_user_client
+from .common import RANCHER_CLEANUP_CLUSTER
+from .common import USER_TOKEN
+from .common import validate_cluster
+from .common import delete_node
+from .test_rke_cluster_provisioning import HOST_NAME
+from .test_rke_cluster_provisioning import random_name
+from .test_rke_cluster_provisioning import rke_config
+from .test_rke_cluster_provisioning import random_test_name
+from .test_rke_cluster_provisioning import get_custom_host_registration_cmd
+
+from lib.aws import AmazonWebServices
+import multiprocessing
+
+
+K8S_VERSION_URL = "/settings/k8s-versions-current"
+PLUGIN = ["calico", "canal", "flannel", "weave"]
+CLUSTER_LIST = []
+NODE_COUNT_KDM_CLUSTER = \
+    int(os.environ.get("RANCHER_NODE_COUNT_KDM_CLUSTER", 4))
+
+
+@pytest.mark.run(order=1)
```

Comment:
There is no need for order, since we have only 1 test .

Reply:
Removed it